suite-native/connection-status/src/OfflineBanner.tsx

```diff
@@ -16,7 +16,7 @@ const containerStyle = prepareNativeStyle(utils => ({
 const contentStyle = prepareNativeStyle<{ topSafeAreaInset: number }>(
     (utils, { topSafeAreaInset }) => ({
         marginTop: topSafeAreaInset,
-        paddingTop: utils.spacings.small,
+        paddingTop: utils.spacings.sp8,
         paddingBottom: 12,
```

Comment:
Since the spacing 12 was not present until now, the codebase is full of hardcoded values of 12. Could you please replace it everywhere 🙏.

```suggestion
        paddingBottom: utils.spacings.sp12,
```

Reply:
Yep, it's in my TODO list. 😉 Will handle it in a follow-up PR.